.github/workflows/pr_basic_compilation_check.yml

```diff
@@ -35,6 +35,14 @@ jobs:
           source /root/.bashrc && make && make install && make clean &&
             make SGX=1 && make clean && make OCCLUM=1
 
+      - name: Compile "rats-tls"
+        if: ${{ !contains(matrix.sgx, 'alinux2') }}
```

Comment:
Please change `if: ${{ !contains(matrix.sgx, 'alinux2') }}` to `if: ${{ !contains(matrix.tag, 'alinux2') }}`